scripts/compile_cost_assumptions.py

```diff
@@ -1404,6 +1407,51 @@ def carbon_flow(costs,year):
 
     return costs
 
+def add_egs_data(data):
+    """
+    Adds data of enhanced geothermal systems.
+
+    Data taken from Aghahosseini, Breyer 2020: From hot rock to useful energy...
+    
+    """ 
+    parameters = ["CO2 intensity", "lifetime", "VOM", "efficiency", "investment", "FOM"]
+    techs = ["geothermal"]
+    multi_i = pd.MultiIndex.from_product([techs, parameters])
+    geoth_df = pd.DataFrame(index=multi_i, columns=data.columns)
+    years = [col for col in data.columns if isinstance(col, int)]
+
+    # lifetime
+    geoth_df.loc[("geothermal", "lifetime"), years] = 30 #years
+    geoth_df.loc[("geothermal", "lifetime"), "unit"] = "years"
+    geoth_df.loc[("geothermal", "lifetime"), "source"] = source_dict["Aghahosseini2020"]
+
+    # co2 emissions
+    geoth_df.loc[("geothermal", "CO2 intensity"), years] = 0.12 # tCO2/MWh
+    geoth_df.loc[("geothermal", "CO2 intensity"), "unit"] = "tCO2/MWh"
+    geoth_df.loc[("geothermal", "CO2 intensity"), "source"] = source_dict["Aghahosseini2020"]
+    geoth_df.loc[("geothermal", "CO2 intensity"), "further description"] = "Likely to be improved; Average of 85 percent of global egs power plant capacity"
+
+    # investment, VOM, efficiency, FOM
+    geoth_df.loc[[
+        ("geothermal", "investment"), ("geothermal", "VOM"),
+        ("geothermal", "efficiency"), ("geothermal", "FOM")
+    ], "unit"] = "tbd"
+    geoth_df.loc[[
+        ("geothermal", "investment"), ("geothermal", "VOM"),
+        ("geothermal", "efficiency"), ("geothermal", "FOM")
+    ], "source"] = source_dict["Aghahosseini2020"]
+    geoth_df.loc[[
+        ("geothermal", "investment"), ("geothermal", "VOM"),
+        ("geothermal", "efficiency"), ("geothermal", "FOM")
+    ], years] = np.nan
```

Comment:
I would suggest not to add the costs here with nan value. You can then only add them in the sector coupling model in your script. We do something similar for the costs of building retrofitting, since they do depend on the building stock in the specific region.

Reply:
Fair enough, not added anymore, see comment below :)